lang/python/python3-speedtest-cli/Makefile

```diff
@@ -0,0 +1,36 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python3-speedtest-cli
+PKG_VERSION:=2.1.2
+PKG_RELEASE:=1
+
+PYPI_NAME:=speedtest-cli
+PKG_HASH:=cf1d386222f94c324e3125ba9a0d187e46d4a13dca08c023bdb9a23096be2e54
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/speedtest-cli-$(PKG_VERSION)
+
+PKG_MAINTAINER:=Jaymin Patel <jem.patel@gmail.com>
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+include ../pypi.mk
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-speedtest-cli
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Command line interface for testing internet bandwidth using speedtest.net
+  URL:=https://github.com/sivel/speedtest-cli
+  DEPENDS:=+python3-setuptools
```

Comment:
Would depending on python3-pkg-resources be enough, instead of depending on python3-setuptools? I'm guessing it needs pkg-resources because it sets up [console scripts](https://github.com/sivel/speedtest-cli/blob/v2.1.2/setup.py#L71-L74), but I don't see where it needs the full setuptools during runtime.

Also, can you add a dependency on either python3 or python3-light? I know there is already an indirect dependency through python3-setuptools/python3-pkg-resources, but this package does directly depend on Python being present.

Reply:
Yes, you are correct. Let me update and test.

Reply:
Fixed. replaced dependency with python3-pkg-resources